A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml

```diff
@@ -0,0 +1,165 @@
+<ModelBehaviors>
+
+    <!--
+    Template for the AP1/AP2 master button on the FCU.
+
+    Should probably be replaced by a generic template at some point.
+    -->
+    <Template Name="FBW_Airbus_Autopilot_Template">
```

Comment:
We should probably call this `FBW_Airbus_Autopilot_Master`.

Reply:
agree